app/src/mainmain/java/pl/targosz/invoicing/App.java

```diff
@@ -4,6 +4,7 @@
 package pl.targosz.invoicing;
 
 public class App {
```

Comment:
![image](https://user-images.githubusercontent.com/17989854/147677435-94198ff8-0c68-4a83-8bdf-fc68e3cacf69.png)

A to co za przejęzyczenie? :P

Reply:
Bo było polecenie żeby zrefactorować, i pomyslalem zeby zmienic nazwę :D

Reply:
@mtszpater pozdrów odemnie proszę Mentora Marczaka, ciekawe czy o mnie pamięta xD